File: Codechef using C-C++/reverse number.cxx

#include <iostream>
	using namespace std;

int main()
{
	int n, rem;
	int rev = 0;
cin>>n;
	while (n > 0)
	{
		rem=n%10;
		rev=rev*10+rem;
		n=n/10;
	}
	cout <<rev;
	return 0;
}

File: Random/ReverseNumber.java

public class ReverseNumber {
    public static void main(String[] args) {
        System.out.println(reverse1(123456789));
    }
    static int reverse1(int n){
        int rev=0;
        int rem;
        while(n>0){
            rem=n%10;
            n=n/10;

            rev=10*rev+rem;
        }
        return rev;
    }
}
